lib/svg_sprite/svg_sprite.rb

```diff
@@ -0,0 +1,218 @@
+module SvgSprite
```

Comment:
https://freelancing-gods.com/2017/07/27/friendly-frozen-string-literals.html
# frozen_string_literal: true

We should probably adopt this globally to be honest